From 337576a387170848732f823bc8729dc841e90213 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:09:13 -0400
Subject: [usb] cdc-acm: remove redundant usb_mark_last_busy

Message-id: <1410887463-20674-482-git-send-email-dzickus@redhat.com>
Patchwork-id: 95549
O-Subject: [RHEL7 PATCH 481/591] USB: cdc-acm: remove redundant usb_mark_last_busy
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 308fee18e0f06215b47b54a2b254bfaf55527bdd
Author: Johan Hovold <jhovold@gmail.com>
Date:   Mon May 26 19:23:50 2014 +0200

    USB: cdc-acm: remove redundant usb_mark_last_busy

    There's no need to call usb_mark_last_busy after having increased the PM
    counter in write(). The device will be marked busy by USB core when the
    PM counter is balanced in the completion handler.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/class/cdc-acm.c b/drivers/usb/class/cdc-acm.c
index 3c7cfac..8f654ce 100644
--- a/drivers/usb/class/cdc-acm.c
+++ b/drivers/usb/class/cdc-acm.c
@@ -690,7 +690,6 @@ static int acm_tty_write(struct tty_struct *tty,
   spin_unlock_irqrestore(&acm->write_lock, flags);
   return count;
  }
- usb_mark_last_busy(acm->dev);
 
  stat = acm_start_wb(acm, wb);
  spin_unlock_irqrestore(&acm->write_lock, flags);
-- 
1.7.1